thelastinuit/21-aug-triple_trouble.exs

```diff
@@ -0,0 +1,18 @@
+# https://www.codewars.com/kata/5704aea738428f4d30000914
+
+defmodule Codewars.WeirdString do
+  def triple_trouble(one, two, three) do
+    # For fancy chars, use graphemes. it's slow, though.
+    chars_one = one |> String.codepoints
```

Comment:
I thought this was sort of terrible but seeing other solutions doesn't look so terrible.